peripherals/timer/timer_13_16_comcen.yaml

```diff
@@ -2,7 +2,7 @@
 #
 # SPDX-License-Identifier: MIT OR Apache-2.0
 
-"TIMER1[3-6]":
-  CHCTL[01]_Output:
+"TIMER1[3456]":
```

Comment:
From what I read in svdtools, ranges are not supported. This means it was trying to match "TIMER13", "TIMER1-" and "TIMER16" before :)

Reply:
Are you sure? It seemed to be working fine before, and the stm32-rs repository is using it, e.g. in https://github.com/stm32-rs/stm32-rs/blob/master/peripherals/tim/tim_basic.yaml.

Reply:
If it uses [this svdtools](https://github.com/stm32-rs/svdtools) I don't see any way to use ranges. The parsing code doesn't seem to handle it (https://github.com/stm32-rs/svdtools/blob/074c31f67938ae21efeb2c74fe01260d7e4f55ff/src/patch/mod.rs#L486) and the documentation has a very small set of tokens (https://github.com/stm32-rs/svdtools#name-matching).
It was not working when I was using it, e.g. `TIM[1-58]` was matching TIM1, TIM5 and TIM8 (so was not reporting an error not finding TIM- because it only needs one match to work), but not TIM2

Reply:
We're using the Python version of svdtools still, so maybe it has more functionality? It does look like there is a bug here though, so I'll have a proper look.

Reply:
I just tried building locally, and I get exactly the same output in the `svd/*.svd.patched` files with or without your change to stop using ranges, so I'm pretty sure the `svd patch` is handling them properly as expected. Removing my `_derivedFrom` fixes does change the output, so we don't want to change that.

I'm not sure why it's not working for you. How are you trying to build it, just using `make`? What version of `svdtools` do you have installed, and did you install from `pip` or somewhere else?

Reply:
Oooh, I was using the rust version. The python version uses braceexpand which handles these kind of things. I'll fix that on my machine and try to rollback the other ones I had unrolled in previous versions too then :)